force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +153,8 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> getCases = [SELECT Origin FROM Case WHERE Origin NOT IN ( 'Web', 'Email', '' )];
+        return getCases;
```

Comment:
For performance purposes, although it may be not as intuitive, you will want to get used to writing IN rather than NOT IN. This does require you to know all of the values you do want to filter on, which is not always possible and can reduce code readability. But depending on the amount of records you are querying, it may be important for query selectivity.